cosmos/cache.py

```diff
@@ -230,9 +230,12 @@ def _create_folder_version_hash(dir_path: Path) -> str:
         filepaths.extend(paths)
 
     for filepath in sorted(filepaths):
-        with open(str(filepath), "rb") as fp:
-            buf = fp.read()
-            hasher.update(buf)
+        try:
+            with open(str(filepath), "rb") as fp:
+                buf = fp.read()
+                hasher.update(buf)
+        except FileNotFoundError:
+            logger.debug(f"Skipping invalid path {filepath}")
```

Comment:
It may be outside of the scope of Cosmos to clean up the users' local filesystem, especially since we're not creating it. Should we change debug to warning and clarify that this may be a symbolic link pointing to a no longer existing file?

Reply:
yes, that makes sense to add a warning and asking to check the user the existence of the file pointed by the symlink.

Reply:
Thanks, @pankajkoti , changes in 40d1909ecdcd5be16d57dab133a5140d98d29e8e